js/packages/openinference-instrumentation-openai/src/instrumentation.ts

```diff
@@ -409,6 +410,24 @@ function getLLMInputMessagesAttributes(
   }, {} as Attributes);
 }
 
+/**
+ * Converts each tool definition into a json schema
+ */
+function getLLMToolsJSONSchema(
+  body: ChatCompletionCreateParamsBase,
+): Attributes {
+  if (!body.tools) {
+    // If tools is undefined, return an empty object
+    return {} as Attributes;
+  }
+  return body.tools.reduce((acc, tool, index) => {
+    const toolJsonSchema = JSON.stringify(tool);
```

Comment:
nit use safelyJSONStringify from @openinference/core

Reply:
agreed, always good to be safe

Reply:
Done